themes/bootstrap3/templates/record/prev-next.phtml

```diff
@@ -0,0 +1,52 @@
+<nav aria-label="<?=$this->transEsc('pagination_navigation')?>">
```

Comment:
There is no translation defined for pagination_navigation in en.ini (unless I'm missing something).

Reply:
Is this meant to be pagination_label, or is it supposed to be something different?